### PR TITLE
VITIS-3632 xbutil/xbmgmt system monitor like support (xbtop)

### PR DESCRIPTION
**Pull request by AShivangi**
#### Problem solved by the commit
As we are removing the support for the old tools, this is the replacement for xbutil top -> `xbtop`

#### How problem was solved, alternative solutions (if any) and why they were rejected
xbtop is python based tool
`/opt/xilinx/xrt/bin`
 |-`xbtop`
`/opt/xilinx/xrt/python`
 |- `ReportPower.py`
 |- `ReportDynamicRegions.py`
 |- `ReportMemory.py`
 |- `XBUtil.py`

#### What has been tested and how, request additional testing if necessary
Built and installed to check the installation.
Executed from Debug and release directories while running validate parallelly 

#### Documentation impact (if any)
We should update the github.io for xbtop, but will do that after review from PRT

### CODE REVIEW

**Review by rozumx — APPROVED** [review]
Approved.  Thank you adding this functionality.

**Comment by AShivangi** [comment]
retest this please

**Comment by sonals** [comment]
Should we add this to the built in test like xbutil and xbmgmt? That will ensure that all the basic python binding is working correctly.

File: src/runtime_src/core/tools/xbtop/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+message(${LINUX_FLAVOR} ${LINUX_VERSION})
+string(SUBSTRING "${LINUX_VERSION}" 0 1 major_version)
+
+if (${LINUX_FLAVOR} STREQUAL "Amazon")
+  set (PYTHON_3 FALSE)
+elseif ((${LINUX_FLAVOR} STREQUAL "CentOS") AND (${major_version} STREQUAL "7"))
+  set (PYTHON_3 FALSE)
+endif ()
+
+if (${PYTHON_3})
```
Comment (eshwar-xilinx) [inline src/runtime_src/core/tools/xbtop/CMakeLists.txt:14]: Hi @AShivangi  ,
The logic seems like install() for xbtop are excluded if distro is CentOS 7.x
Will this affect behavior of `xbtop` on CentOS 7.x ? Or this change is due to python version being used on Centos ?

Comment (AShivangi) [reply src/runtime_src/core/tools/xbtop/CMakeLists.txt:14]: I'll discuss this with you offline

**Comment by dayeh-xilinx** [comment]
retest this please. re-enabled PR branch.

**Pull request merged**